lib/sensors/tof.cpp

```diff
@@ -48,40 +48,32 @@ core::Result TimeOfFlight::initialise()
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[6], 0x03);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[7], 0x02);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[8], 0x05);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[9], 0xce);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[10], 0x03);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[11], 0xf8);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[12], 0x00);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[13], 0x3c);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[14], 0x00);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[15], 0x3c);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[16], 0x09);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[17], 0x09);
-    // i2c_->writeByteToRegister(device_address_, 0x0198, 0x01);
-    // i2c_->writeByteToRegister(device_address_, 0x01b0, 0x17);
-    // i2c_->writeByteToRegister(device_address_, 0x01ad, 0x00);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[18], 0x05);
-    // i2c_->writeByteToRegister(device_address_, 0x0100, 0x05);
-    // i2c_->writeByteToRegister(device_address_, 0x0199, 0x05);
-    // i2c_->writeByteToRegister(device_address_, 0x01a6, 0x1b);
-    // i2c_->writeByteToRegister(device_address_, 0x01ac, 0x3e);
-    // i2c_->writeByteToRegister(device_address_, 0x01a7, 0x1f);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[19], 0x00);
-
+    constexpr std::uint8_t data[] = {0xfd,
+                                     0x01,
+                                     0x03,
+                                     0x02,
+                                     0x01,
+                                     0x03,
+                                     0x02,
+                                     0x05,
+                                     0xce,
+                                     0x03,
+                                     0xf8,
+                                     0x00,
+                                     0x3c,
+                                     0x00,
+                                     0x3c,
+                                     0x09,
+                                     0x09,
+                                     0x05,
+                                     0x00};
+    for (int i = 0; i < 20; i++) {
+      i2c_->writeByteToRegister(device_address_, array_return_addresses[1], data[i]);
+    }
     i2c_->writeByteToRegister(device_address_, kSystemFreshOutOfReset, 0);
 
     // Recommended : Public registers
     i2c_->writeByteToRegister(device_address_, kSystemModeGpioOne, 0x10);
-    // i2c_->writeByteToRegister(device_address_, kReadoutSamplingPeriod, 0x30);
```

Comment:
Again, this was a 16-bit register which would cause build failure, not a line which needed to be removed. It should be added back in